test/integration/http-test.js

```diff
@@ -608,6 +608,73 @@ describe('HTTP APIs', function () {
         })
     })
 
+    describe('content-type-based file extensions', () => {
+      // ensure the container exists
+      before(() =>
+        server.post('/post-tests/')
+          .send(postRequest1Body)
+          .set('content-type', 'text/turtle')
+      )
```

Comment:
Make sure to have an `after` hook which cleans up!

Reply:
Good point, it's further down.